src/modules/Laboratorio/Listar/index.js

```diff
@@ -1,41 +1,39 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 
 import Menu from '../../../utils/components/Menu';
 import Title from '../../../utils/components/Title';
 import Toolbar from '../../../utils/components/Toolbar';
 import Laboratorio from './components/Laboratorio';
+import api from '../../../services/api';
 import './styles/index.css';
 
-function Listar() {
-  const labs = {
-    lab01: {
-      id: '01',
-      name: 'Laboratorio 01',
-      location: 'DCOMP Antigo',
-      status: 'Livre',
-      capacity: '40',
-    },
-    lab02: {
-      id: '02',
-      name: 'Laboratorio 02',
-      location: 'DCOMP Antigo',
-      status: 'Livre',
-      capacity: '20',
-    },
-  };
-  const quantidade = `${Object.keys(labs).length} itens`;
+const Listar = () => {
+  const [state, setState] = useState([]);
+
+  useEffect(() => {
+    async function fetchData() {
+      const res = await api.get('/laboratory');
+      setState(await res.data);
```

Comment:
Esse `await` é redundante. A promessa (_promise_) do _axios_ já foi finalizada com `àwait` e o `res.data` é um objeto, não uma promessa.

```suggestion
      setState(res.data);
```